app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -576,6 +576,9 @@ class BrowserTabFragment : Fragment(), FindListener {
                 useWideViewPort = true
                 builtInZoomControls = true
                 displayZoomControls = false
+
+                // explicitly disable database to try protect against Magellan WebSQL/SQLite vulnerability
```

Comment:
Does it make sense to add this to it's own method (could be an extension)? Something explicit like protectAgainstMagellanVulnerability(). I always worry that comments like this become ambiguous over time. For example, imagine we add two more storage settings underneath this one and it is no longer clear if the comment applies to just this line of code or all three.